browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -40,6 +37,14 @@
 #include "content/public/browser/reload_type.h"
 #include "content/public/browser/web_contents.h"
 
+#if BUILDFLAG(IS_ANDROID)
+#include "base/containers/contains.h"
```

Comment:
what is that include for?

Reply:
It's for https://github.com/brave/brave-core/blob/ce618de91891a88f6d712fb3dd2de5f345aed04c/browser/speedreader/speedreader_tab_helper.cc#L451, it needs the header file.

Reply:
I think it is because `brave/browser/ui/brave_browser_window.h` also includes this file so it was not needed before. But I can make it always includes this file.